Origin: http://git.xfce.org/xfce/thunar/commit/?id=c9712b835be90127ef4af1ac1b1fd645081a646a
Description: Fix "Home" and "Open Parent" icons going missing from toolbar
Bug: https://bugzilla.xfce.org/show_bug.cgi?id=10179

---
 thunar/thunar-window.c |    1 +
 1 file changed, 1 insertion(+)

--- a/thunar/thunar-window.c
+++ b/thunar/thunar-window.c
@@ -1821,6 +1821,7 @@ thunar_window_install_location_bar (Thun
           /* setup the toolbar for the location bar */
           window->location_toolbar = gtk_ui_manager_get_widget (window->ui_manager, "/location-toolbar");
           g_object_get (G_OBJECT (window->preferences), "misc-small-toolbar-icons", &small_icons, NULL);
+          gtk_toolbar_set_style (GTK_TOOLBAR (window->location_toolbar), GTK_TOOLBAR_ICONS);
           gtk_toolbar_set_icon_size (GTK_TOOLBAR (window->location_toolbar),
                                      small_icons ? GTK_ICON_SIZE_SMALL_TOOLBAR : GTK_ICON_SIZE_LARGE_TOOLBAR);
           gtk_table_attach (GTK_TABLE (window->table), window->location_toolbar, 0, 1, 1, 2, GTK_EXPAND | GTK_FILL, GTK_FILL, 0, 0);
